src/open-uri.c

```diff
@@ -191,8 +191,16 @@ launch_application_with_uri (const char *choice_id,
 
   if (is_sandboxed (info))
     {
+      g_autoptr(GError) error = NULL;
+
       g_debug ("registering %s for %s", uri, choice_id);
-      ruri = register_document (uri, choice_id, FALSE, writable, NULL);
+      ruri = register_document (uri, choice_id, FALSE, writable, &error);
+      if (ruri == NULL)
+        {
+          g_warning ("Error registering %s for %s", uri, choice_id, error->message);
+          g_clear_error (&error);
```

Comment:
Nitpick: The `g_clear_error()` is technically not necessary here, since you’ve used `g_autoptr()` above.

Reply:
Same thing, sloppy mistake, sorry

Reply:
(I’d say it’s a stylistic issue rather than a mistake!)